tests/integration/Owncloud/OcisPhpSdk/ShareGetSharedWithMeTest.php

```diff
@@ -59,9 +59,6 @@ public function setUp(): void
     public function testGetAttributesOfReceivedShare(): void
     {
         $this->fileToShare->invite($this->einstein, $this->editorRole);
-        /**
-         * @var ShareReceived $receivedShare
-         */
         $receivedShare = $this->getSharedWithMeWaitTillShareIsAccepted($this->einsteinOcis)[0];
```

Comment:
Why removing this?
My IDE does does do autocomplete with the reference, but not without
![grafik](https://github.com/owncloud/ocis-php-sdk/assets/2425577/b5c31e16-cecb-494b-bb85-2be9f1d3393f)

Reply:
But autocomplete works for me with PHPstorm.